src/main/java/org/occiware/tinom/aggregators/AIWA.java

```diff
@@ -0,0 +1,50 @@
+
+package org.occiware.tinom.aggregators;
+
+/**
+ *
+ * @author Mehdi Ahmed-Nacer
```

Comment:
I think you got a warning from Checkstyle because we expect this tag to follow the convention `@author name - company`.